### PR TITLE
Completed email modal with email populated

### PR DESCRIPTION
**Pull request by Itav8**

### CODE REVIEW

File: frontend/src/pages/Guests/GuestInvite.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+      const inviteResponse = await fetch(inviteGuest, fetchConfig);
+
+      if (inviteResponse.ok) {
+        setEmailForm({
+          email: "",
+          subject: "",
+          message: "",
+        });
+
+        props.onSubmit;
```
Comment (misodope) [inline frontend/src/pages/Guests/GuestInvite.tsx:64]: Not invoked

File: frontend/src/pages/Guests/GuestInvite.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+      [inputName]: value,
+    });
+  };
+
+  const handleInvite = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    const inviteGuest = `${import.meta.env.VITE_API_URL}/guest/invite`;
+    const messageData = {
+      email: props.initialValues,
```
Comment (misodope) [inline frontend/src/pages/Guests/GuestInvite.tsx:41]: The initial value is just a string?

File: frontend/src/pages/Guests/Guests.tsx [hunk 3]
```diff
@@ -240,6 +215,22 @@ export const Guests = () => {
...
       ) : null}
+      {isEmailModalOpen ? (
+        <Modal
+          open={isEmailModalOpen}
+          onClose={() => {
+            setIsEmailModalOpen(false);
+          }}
+        >
+          <GuestInvite
+            initialValues={selectedEmail}
```
Comment (misodope) [inline frontend/src/pages/Guests/Guests.tsx:226]: I think if it's not going to be an object, we can maybe rename this prop from `initialValues` to maybe just `email` or `selectedEmail`

File: frontend/src/pages/Guests/GuestInvite.tsx [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+    <div>
+      <h1>Email</h1>
+      <form className="guest-invite__form" onSubmit={handleInvite}>
+        <div>
+          <label htmlFor="email">Email:</label>
+          <input
+            type="email"
+            id="email"
+            name="email"
+            value={props.initialValues}
```
[the remark this answers is not in the document]
Comment (Itav8) [reply frontend/src/pages/Guests/GuestInvite.tsx:83]: I think this is backwards

**Review by misodope — APPROVED** [review]
lgtm

**Pull request merged**